### ISSUE TITLE
textblob library

### ISSUE THREAD

**Issue by ghost** [EricSchles/traffickingGrab#5]
I believe that in order for this be used by people, this project should be written in plain vanilla. However, I've found one library so far (`textblob`) that I've had to install. Even then, I have come across trouble using it to be compatible with your scripts (which isn't the reason I'm writing this, but you see the point). I would be happy to help find a way around this to make this easier for everyone if that means either writing an installation script (which I have attempted with the file below) or modifying the code to support text classification without the need of `textblob`.

[setup.sh.zip](https://github.com/EricSchles/traffickingGrab/files/4843261/setup.sh.zip)

**Comment by MattMcFahn** [comment]
Hey Zach,

The python ecosystem usually requires users to install "non-vanilla" packages and modules. A number of packages, such as textblob, are not part of the standard python distribution, but most users wishing to do NLP will install textblob in the environment they're working with - there simply isn't an option to do such work without textblob (or a similar module).

Tl;dr - Not using textblob isn't feasible (similar for a number of other packages).

The way this is usually managed in a python based repo is via a "requirements.txt" file listing the required dependencies (allowing for a user to easy install with a command such as pip install requirements.txt).

I'll produce such a file this evening, and producing installation documentation is something to get done in the short term too.

P.S. Given work is happening on your fork of this repo, rather than this one, would it make more sense to move the conversation & opening of issues there?

**Comment by ghost** [comment]
I thought it would be a good idea to put this issue here, so people can benefit from it besides myself.
